src/Controls/src/Core/Platform/Windows/VisualElementTracker.cs

```diff
@@ -555,12 +555,15 @@ void OnPointerCanceled(object sender, PointerRoutedEventArgs e)
 
 		void OnPointerExited(object sender, PointerRoutedEventArgs e)
```

Comment:
Why do we have this duplicate code? GesturePlatformManager AND VisualElementTracker?

Reply:
Reverted the changes in VisualElementTracker. Are not necessary, we use now the GesturePlatformManager and the Tracker is obsolete from .NET 7.